### DISCUSSION TITLE
Repeat values in a dropdown multi select?

### DISCUSSION THREAD

**Discussion by RustyBower** [fomantic/Fomantic-UI#1984] [Collections (form, grid, menu, message, table, ...)] [answered]
Is there a good way to get multiple values in a dropdown multi select?

For example, if I am asking people what pets they have, they could select Dog Dog Dog Cat

**Comment by lubber-de** [comment, marked as answer]
Please check the multiple selection variant of dropdown
https://fomantic-ui.com/modules/dropdown.html#multiple-selection

**Comment by RustyBower** [comment]
Sorry I may not have been clear, in the multi search selection drop down example, I want to be able to add Alabama multiple times. Is that possible?

**Reply by lubber-de** [reply]
Yes, just make sure each option has it's unique value
See https://jsfiddle.net/weLqh4m3/

**Comment by RustyBower** [comment]
This works. Thanks!
